rewrite-maven/src/test/java/org/openrewrite/maven/RemoveRepositoryTest.java

```diff
@@ -98,6 +98,54 @@ void removeSimpleRepoWithUrlOnly() {
     @Test
     void removePluginSimpleRepoWithIdAndUrl() {
 
+        rewriteRun(
+          spec -> spec.recipe(new RemoveRepository("myRepo", "http://myrepo.maven.com/repo", null, null,
+            null, null, null,
+            null, null, null)),
+          pomXml(
+            """
+              <project>
+                <groupId>com.mycompany.app</groupId>
+                <artifactId>my-app</artifactId>
+                <version>1</version>
+                <pluginRepositories>
+                  <pluginRepository>
+                    <id>myRepo</id>
+                    <url>http://myrepo.maven.com/repo</url>
+                  </pluginRepository>
+                  <pluginRepository>
+                    <id>identicalRepo</id>
+                    <url>http://myrepo.maven.com/repo</url>
+                  </pluginRepository>
+                </pluginRepositories>
+                 <repositories>
+                  <repository>
+                    <id>myRepo</id>
+                    <url>http://myrepo.maven.com/repo</url>
+                  </repository>
+                </repositories>
+              </project>
+              """,
+            """
+             <project>
+               <groupId>com.mycompany.app</groupId>
+               <artifactId>my-app</artifactId>
+               <version>1</version>
+               <pluginRepositories>
+               </pluginRepositories>
+                <repositories>
+               </repositories>
```

Comment:
Should we have the option to allow users to pick? Or is it ok to remove from both as a default?

Reply:
I think it's fine to remove both, or at least until a compelling argument to introduce a parameter is made in a new issue. The use cases I see for removing repositories is typically when repositories are decommissioned; in that case it makes sense to remove both the regular repository and the plugin repository at the same time.